### DISCUSSION TITLE
Bleachers on football field

### DISCUSSION THREAD

**Discussion by mgreenoo** [OpenOrienteering/mapper#1874] [Questions & Answers]
Looking for advice about bleachers on a football field.
What's the best symbol for bleachers?
The widest footprint for stairs is only 4m so it doesn't really work here:
![image](https://user-images.githubusercontent.com/79408443/108628148-3cf27000-7427-11eb-8e9f-c4f30df42670.png)
Many thanks!

**Comment by ollesmaps** [comment]
You can draw in ground plan with symbols Step or edge of paved area, Paved area Impassable fence, etc.
I would suggest to raise such questions rather at [Orienteering Mappers Int. ](https://www.facebook.com/groups/485564718218028/)

**Reply by mgreenoo** [reply]
Thanks so much. I will check out the group there.

**Comment by mlerjen** [comment]
https://www.facebook.com/groups/485564718218028/permalink/3294800343961104/ contains some discussion on how to represent bleachers.

**Reply by mgreenoo** [reply]
Thanks so much---the facebook page looks like a good spot to post questions.
